metadata/src/test/java/org/apache/kafka/controller/ConfigurationControlManagerTest.java

```diff
@@ -72,6 +73,14 @@
             define("ghi", ConfigDef.Type.BOOLEAN, true, ConfigDef.Importance.HIGH, "ghi"));
     }
 
+    static class ExampleConfigurationValidator implements ConfigurationValidator {
+        static final ExampleConfigurationValidator INSTANCE = new ExampleConfigurationValidator();
+
+        @Override
+        public void validate(ConfigResource resource, Map<String, String> config) {
+        }
```

Comment:
Did you plan to implement this? If you not, you can use the trick you are doing in other places. E.g.
```java
static ConfigurationValidator NOOP_VALIDATOR = (_, __) -> {};
```

Reply:
ok, I'll do that